### PR TITLE
Mina deployment update

### PR DESCRIPTION
**Pull request by jhulme**
This update includes:

1) The re-write of the db:configure task into shell script. This mirrors the changes already applied to the mina deploy file for the API. 

2) Addition of a user input prompt prior to deploying a fresh database instance.

3) Unbundling of the db:setup command into db:create migrate and seed. This removes the db:schema:load command, essentially adding another safety catch to deployment.

This pull request now supercedes the mar16 branch.

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -25,6 +25,10 @@
 #   set :port, '30000'     # SSH port number.
 set :forward_agent, true     # SSH forward_agent.
 
+#Database configuration check
+set :dbconncheck, lamdba { "#{connexchk}" }
```
Comment (samnissen) [inline config/deploy.rb]: What does this do? Does it have something to do with https://github.com/samnissen/polariscope_2/pull/62#discussion_r58859795

**Comment by jhulme** [comment]
Initially yes - I dropped the function as it didn't have the desired effect and have forgotten to remove this line.

**Comment by jhulme** [comment]
This line is removed now so you can press on with the merge. If you still think we require the additional safety of a database connectivity check, I think it would be best to raise as a separate ticket as adding functionality was not straightforward.

**Comment by samnissen** [comment]
I'm sorry, @jhulme, you're going to kill me -- line 121 has double in the file path ("/config/config/").

**Comment by jhulme** [comment]
Hi @samnissen - no problem, that should be correct now.

**Pull request merged**